pkg/conf/config.go

```diff
@@ -16,6 +16,7 @@ const (
 	gcsKey              = "key.json"
 )
 
+// Parameters is map which contains storage params
```

Comment:
Nitpick: Effective Go recommends doc strings be full sentences. Could you reword this slightly?
`// Parameters is a map which contains storage params.`